src/timeline.js

```diff
@@ -9,11 +9,19 @@ import interact from 'interactjs';
 import _ from 'lodash';
 
 import {sumStyle, pixToInt, intToPix} from 'utils/common';
-import {rowItemsRenderer, getTimeAtPixel, getNearestRowHeight, getMaxOverlappingItems} from 'utils/itemUtils';
+import {
+  rowItemsRenderer,
+  getTimeAtPixel,
+  getNearestRowHeight,
+  getMaxOverlappingItems,
+  getDurationFromPixels
+} from 'utils/itemUtils';
 import {groupRenderer} from 'utils/groupUtils';
-
 import Timebar from 'components/timebar';
+
 import './style.css';
+import {isThisSecond} from 'date-fns';
```

Comment:
What's this for? Is it a moment alternative thing?

Reply:
i thought you added it in, i'll remove